deepspeed/runtime/lr_schedules.py

```diff
@@ -727,6 +736,13 @@ def __init__(self,
         self.max_lrs = self._format_param(self.optimizer, warmup_max_lr, "max_lr")
         self.delta_lrs = [big - small for big, small in zip(self.max_lrs, self.min_lrs)]
         self.warmup_num_steps = max(2, warmup_num_steps)
+        # Currently only support linear and log function
+        if warmup_type not in {'log', 'linear'}:
```

Comment:
Can we avoid hardcoded constants 'log' and 'linear' with global vars, such as WARMUP_LOG_RATE and WARMUP_LINEAR_RATE? Or any names that seem reasonable? Thanks.

Reply:
@tjruwase Thanks for your suggestion. I've just replaced them with global variables